### ISSUE TITLE
Evaluate future video transport with Salsify for performance/reliability

### ISSUE THREAD

**Issue by heyakyra** [vector-im/matrix-video-chat#175]
This would be one way to beat the performance of mainstream videoconferencing applications. 

It outperforms the rest: https://snr.stanford.edu/salsify/

**Comment by robertlong** [comment]
I don't think this is something we can do without browser support. When we start working with SFUs we'll need to think a bit more about codecs, but we'll likely need to go with the lowest common denominator for the best support.

Thanks for sharing this though, it's an interesting idea to look into in the future!
